github/ipfs.yml

```diff
@@ -5809,12 +5772,10 @@ repositories:
     squash_merge_commit_title: COMMIT_OR_PR_TITLE
     teams:
       admin:
-        - admin
         - ipdx
         - kubo maintainers
-        - w3dt-stewards
-      pull:
-        - github-mgmt stewards
+      maintain:
+        - shipyard
```

Comment:
@lidel WDYT about giving push to repos Go for rainbow and someguy?

Reply:
Probably ok, that group already has "maintain" in boxo.

ps. there is a separate long-term meta-worry in that there is way too many people in https://github.com/orgs/ipfs/teams/repos-go, and if our intention is to limit security risks / access, we should plan to subset that group.